traveler-ios-ui/TravelerKitUI/ViewControllers/OrderResultViewController.swift

```diff
@@ -35,6 +37,19 @@ open class OrderResultViewController: UITableViewController {
         }
     }
 
+    @objc func orderDidCancel (_ note: Notification) {
+        guard let order = note.userInfo?[orderKey] as? Order else {
+            Log("Invalid notification", data: note, level: .error)
+            return
+        }
+
+        guard let index = try? self.orderResult!.update(order) else {
```

Comment:
Dont force unwrap here

Reply:
This is the only comment now that requires attention, the rest is just feedback, Ill leave it upto you if you want to change or not. but force unwrap is dangerous.

To get around this you need some clever braces

I think this will do
`let index = try? (self.orderResult?.update(order))`

OR:

`let result = orderResult, let index = try? result.update(order)`